CMakeLists.txt

```diff
@@ -0,0 +1,121 @@
+project(motion)
+cmake_minimum_required(VERSION 2.8)
+include(CheckIncludeFiles)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+
+execute_process(COMMAND sh "${CMAKE_CURRENT_SOURCE_DIR}/version.sh" OUTPUT_VARIABLE VERSION)
+set(BIN_PATH "${CMAKE_INSTALL_PREFIX}/bin")
+set(sysconfdir "/etc" CACHE STRING "system configuration directory")
```

Comment:
I think this differs from the behaviour of the current setup which defaults to /usr/local/etc - we should probably keep the default as /usr/local/etc ?